kastrubait/shop-NodeJs-AWS
# Task-9 Backend For Frontend

Task: https://github.com/EPAM-JS-Competency-center/cloud-development-course-initial/blob/main/task9-bff-elastic-beanstalk/task.md
Done 06.11.2021 / Deadline 06.11.2021
Score: 7 / 7

**Links:**

product-service: https://qxm1bk977h.execute-api.eu-west-1.amazonaws.com/dev/products
bff-product-service: http://kastrubait-bff-api-dev.eu-west-1.elasticbeanstalk.com/products 
(**using express** - http://kastrubait-bff-express-api-dev.eu-west-1.elasticbeanstalk.com/products)

cart-service: http://kastrubait-cart-api-dev.eu-west-1.elasticbeanstalk.com/api/profile/cart
bff-cart-service: http://kastrubait-bff-api-dev.eu-west-1.elasticbeanstalk.com/cart 
(**using express** - http://kastrubait-bff-express-api-dev.eu-west-1.elasticbeanstalk.com/cart)

**Evaluation criteria**


**Additional tasks**

   _How to test:_
   - _Get products list_
   - _Create new product_
   - _Get products list - result shouldn’t have new product_
   - _Wait more than 2 minutes_
   - _Get products list - result should have new product_
